edge/pkg/edged/edged.go

```diff
@@ -166,11 +167,19 @@ func newEdged(enable bool, nodeName, namespace string) (*edged, error) {
 	if !edgedconfig.Config.RegisterNode {
 		kubeletConfig.RegisterNode = false
 	}
+
+	// set feature gates from initial flags-based config
+	if err := utilfeature.DefaultMutableFeatureGate.SetFromMap(kubeletConfig.FeatureGates); err != nil {
```

Comment:
Without the fix above, the FeatureGate will doesn't work, right?

Reply:
If so, it should be a bug fix? And should be cherry picked to releases

Reply:
> Without the fix above, the FeatureGate will doesn't work, right?

Yes, I will separate it to another bugfix pr.